src/app-bar.jsx

```diff
@@ -314,26 +313,26 @@ const AppBar = React.createClass({
           {children}
       </Paper>
     );
-  },
+  }
 
+  @autobind
```

Comment:
Having a look at https://github.com/callemall/material-ui/pull/1734/files, I would say, let's let react-codemod puts `this`.